### PR TITLE
Add marketplace app guides for compute

### PR DESCRIPTION
**Pull request by aayushmahapatra**
- MongoDB
- Redis
- PostgreSQL
- MySQL
- Grafana
- HashiCorp Vault
- IPFS

### CODE REVIEW

**Review by rekpero — APPROVED** [review]
lgtm

File: components/quick-start.tsx [hunk 1]
```diff
@@ -44,6 +44,12 @@ const QUICK_START_LIST = [
     title: "Server Guide",
     body: "Learn how to deploy your server.",
   },
+  {
+    path: "/marketplace-guide",
+    emoji: "🏪",
+    title: "Marketplace Guide",
+    body: "Deploy pre-installed apps and scale fast.",
```
Comment (shomix) [inline components/quick-start.tsx:51]: hmm, not sure if scale is right here?
Maybe `and develope fast.`?

cc @rekpero

File: pages/marketplace-guide/mongodb.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+You can use the connection string URI to connect to MongoDB.
+The standard URI connection scheme has the following format:
+
+```
+mongodb://[username:password@]host[:port][/[defaultauthdb][?options]]
+```
+
+- `mongodb://` - A required prefix to identify that this is a string in the standard connection format.
+- `username:password` **(optional)** - Authentication credentials passed in App Configuration. If specified, the client will attempt to authenticate the user to the `authSource`.
+- `host[:port]` - The host and port number where the mongod instance is running. Use the `Connection URL` provided by Spheron under **Port Policy Info for Port 27017**.
```
Comment (shomix) [inline pages/marketplace-guide/mongodb.mdx:45]: I would add a not to use the value of the external port

**Review by shomix — APPROVED** [review]

File: pages/marketplace-guide/mongodb.mdx (hunk 2 above)
Comment (shomix) [reply pages/marketplace-guide/mongodb.mdx:45]: Everything looks good. Not sure just if we should mention this about the external port ( for each market place app ), or if the users will now which one to use?

cc @rekpero

File: components/quick-start.tsx (hunk 1 above)
Comment (rekpero) [reply components/quick-start.tsx:51]: yes correct

File: pages/marketplace-guide/mongodb.mdx (hunk 2 above)
Comment (rekpero) [reply pages/marketplace-guide/mongodb.mdx:45]: not sure what you mean here, can you elaborate why you think so?

Comment (aayushmahapatra) [reply pages/marketplace-guide/mongodb.mdx:45]: Discussed with @rekpero. I will update it accordingly.

**Pull request merged**